app/services/apis/os_places/polygon_search_service.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+require "faraday"
+
+module Apis
+  module OsPlaces
+    class PolygonSearchService < Client
+      RESULTS_PER_PAGE = 100
+      MAX_TOTAL_RESULTS = 1000
```

Comment:
Can I just check where this number comes from? Is it that we know OS Places will never return more than 1000 results, or that we would never want to send more than 1000 letters?

Reply:
An arbitrary number I just made up. Need to have a limit of some sort otherwise we could theoretically have a (near) infinite loop if the polygon covered a large enough area.

Reply:
Yeah, we don't want it completely unbounded. Let's keep an eye on it, it's _probably_ fine but I can imagine cases where there are a number of large blocks of flats nearby and we might actually legitimately reach that limit. (And although the user could then add more in a separate request it might be difficult to know which ones had been missed out. But this is definitely an edge case we might not ever need to be concerned with.)